src/sentry/sentry_metrics/indexer/postgres/postgres_v2.py

```diff
@@ -111,7 +113,7 @@ def bulk_record(
                 UseCaseKeyResult(
                     use_case_id=(
                         UseCaseID.TRANSACTIONS
-                        if self._get_metric_path_key(strings.keys()) is UseCaseKey.PERFORMANCE
+                        if metric_path_key is UseCaseKey.PERFORMANCE
                         else UseCaseID.SESSIONS
```

Comment:
don't we have a method to do this mapping somewhere?

Reply:
This actually needs to be changed to
```python3
UseCaseID(db_obj.use_case_id)
if metric_path_key is UseCaseKey.PERFORMANCE
else UseCaseID.SESSIONS
```
When the backfill is complete, but it's not finished by ops yet.

Reply:
Yep, once the backfill is done, we can also merge https://github.com/getsentry/sentry/pull/49095 and this will be updated.

Reply:
Marking this as resolved now